ci-automation/vendor-testing/qemu_update.sh

```diff
@@ -67,14 +72,40 @@ if [ "${CIA_ARCH}" = "arm64" ]; then
     fi
 fi
 
-kola run \
-    --board="${CIA_ARCH}-usr" \
-    --parallel="${QEMU_PARALLEL}" \
-    --platform=qemu \
-    --qemu-bios="${bios}" \
-    --qemu-image=tmp/flatcar_production_image_previous.bin \
-    --tapfile="${CIA_TAPFILE}" \
-    --torcx-manifest="${CIA_TORCX_MANIFEST}" \
-    --update-payload=tmp/flatcar_test_update.gz \
-    --qemu-skip-mangle \
-    cl.update.payload
+query_kola_tests() {
+    shift; # ignore the instance type
+    local arg
+    arg="${*}"
+    if [ "${arg}" != "" ]; then
+      arg="cl.update.payload"
+    fi
+    kola list --platform=qemu --filter "${arg}"
```

Comment:
This could be a bit shorter:

```suggestion
    kola list --platform=qemu --filter "${*:+cl.update.payload}"
```

I always refer to https://wiki.bash-hackers.org/syntax/pe for parameter expansion stuff.

Reply:
Shorter or readable? :D

Reply:
The `shift` would still be needed, or?

Reply:
> Shorter or readable? :D

> The `shift` would still be needed, or?

Oh yeah, shift is still needed. Drat, there goes my "Bash Golf". So, with the `shift` left intact, it will be neither short nor readable. :P